json-spec/examples/planet-sample.json

```diff
@@ -55,30 +55,31 @@
   "id": "20171110_121030_1013",
   "properties": {
     "datetime": "2017-11-10T12:10:30.535417Z",
-    "provider": "Planet PlanetScope 1013",
-    "acquired": "2017-11-10T12:10:30.535417Z",
-    "anomalous_pixels": 0.14,
-    "cloud_cover": 0.23,
-    "columns": 9111,
-    "epsg_code": 32724,
-    "ground_control": true,
-    "gsd": 4,
-    "instrument": "PS2",
-    "item_type": "PSScene4Band",
-    "origin_x": 508263,
-    "origin_y": 8557524,
-    "pixel_resolution": 3,
-    "provider": "planetscope",
-    "published": "2017-11-10T16:11:38Z",
-    "quality_category": "standard",
-    "rows": 4466,
-    "satellite_id": "1013",
-    "strip_id": "902639",
-    "sun_azimuth": 101.8,
-    "sun_elevation": 58.8,
-    "updated": "2017-11-11T11:17:33Z",
-    "usable_data": 0,
-    "view_angle": 1
+    "provider": "Planet",
+    "eo:cloud_cover": 23,
+    "eo:gsd": 4,
+    "eo:sun_azimuth": 101.8,
+    "eo:sun_elevation": 58.8,
+    "eo:off_nadir": 1,
+    "pl:acquired": "2017-11-10T12:10:30.535417Z",
+    "pl:anomalous_pixels": 0.14,
+    "pl:columns": 9111,
+    "pl:epsg_code": 32724,
+    "pl:ground_control": true,
+    "pl:instrument": "PS2",
+    "pl:item_type": "PSScene4Band",
+    "pl:origin_x": 508263,
+    "pl:origin_y": 8557524,
+    "pl:pixel_resolution": 3,
+    "pl:provider": "planetscope",
```

Comment:
Is pl:provider supposed to be different from provider? Wouldn't planetscope be more of a product or sensor name than a provider?

Also, I wouldn't hold anything up for it. But I think we are going to see tons of different date strings per provider. We might want to figure out the published, updated, acquired, start, end, etc. convention at some point.

Reply:
I paused on that one, but I kinda wanted to show that one could just take one's internal fields and map what makes sense and just include the rest. Definitely a redesigned Planet api with STAC and eo profiles in mind would change that. And indeed it'd probably use the eo sensor field. 

I had the same thought about published, updated, acquired as well. Published and updated feel like they should almost be at a WFS level, as any feature might have those fields, and it'd be good if clients could expect a common name. And then some optional time fields in the EO profile seem like they could be useful.

I do think we should pretty soon take a survey of the fields people add in their provider section to the EO profile and just get to common definitions - more optional fields in EO profile. Like rows and columns, origin_x, origin_y, epsg code, etc.